app/src/main/java/com/sdpteam/connectout/profile/ProfileFirebaseDataSource.java

```diff
@@ -52,6 +58,12 @@ public CompletableFuture<Boolean> saveProfile(Profile profile) {
         return finished;
     }
 
+    /**
+     * Retrieves a Profile from firebase using its Id.
+     *
+     * @param uid (String): id of the user to find
+     * @return (CompletableFuture < Profile >): completes to the matching profile Id
+     */
     @Override
     public CompletableFuture<Profile> fetchProfile(String uid) {
```

Comment:
should name that local variable `profileId`, `uid` is outdated from when we started our project